### ISSUE TITLE
I have an issue with forcing RTL after using latest version 5.0.9

### ISSUE THREAD

**Issue by hisham2007** [mikepenz/MaterialDrawer#1058]
Many thanks for great effort. I have a strange issue:
My app targets Arabian users only so I force local to be Arabic.
I was using Material drawer 4.6.4 for long time without issues. you can check my app on market: https://play.google.com/store/apps/details?id=com.tabe3.news

Today, I checked github and found many updates so I updated Material drawer to latest version 5.0.9

After using it, I found Strings in my app are displayed in English. I made sure that I did'nt make change in code except updating material drawer.

the method I use in forcing local:

```java
 String languageToLoad = "ar"; // your language
        Locale locale = new Locale(languageToLoad);
        Locale.setDefault(locale);
        Configuration config = new Configuration();
        //config.locale = locale;
        config.setLocale(locale);
        activity.getBaseContext().getResources().updateConfiguration(config,
                activity.getBaseContext().getResources().getDisplayMetrics());
        activity.getWindow().getDecorView().setLayoutDirection(View.LAYOUT_DIRECTION_RTL);
```

Please advise. 
Thank you,
Hisham

**Comment by hisham2007** [comment]
I think I found the issue. it is related to a bug in support library v23.2.0 . I checked Material drawer versions history and found that you updated support liberary from v23.1.1 to v23.2.0 in Material drawer v5.0.6  So I reverted to Material drawer v5.0.5 and my issue is fixed.

**Comment by mikepenz** [comment]
Oh ok. so it seems this will be fixed in v23.2.1 of the support lib
